src/routes/incidents/table/Utils.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+export const renderImagesCollection = (row) => {
+  return (
+    <div>
+      {row.images.length > 0
+        ? row.images.map((image, index) => {
+            return (
+              <a
+                key={index}
+                href={image}
+                target="_blank"
+                rel="noopener noreferrer"
+                style={{textDecoration: 'none'}}>
+                {`${++index} `}
+              </a>
+            );
+          })
+        : 'N/A'}
+    </div>
+  );
+};
+
+export const renderTitle = (row) => {
+  return (
+    <div>
+      <a
```

Comment:
Creo que react-router-dom ofrece un componente <Link> para hacer esto, creo que con los <a> triggereas un refresh de la página, puede ser?

Reply:
No tengo ni idea, tendria que fijarme. En un rato reviso jjajaja

Reply:
Por lo que dice https://stackoverflow.com/questions/43087007/react-link-vs-a-tag-and-arrow-function es como vos decis Lucas, aun asi yo lo dejaria asi para este Sprint, por que al hacer la modificacion por Link me esta tirando un error 

![Screenshot from 2020-11-02 03-03-50](https://user-images.githubusercontent.com/28553568/97835068-12ce4400-1cb8-11eb-9421-1a10909ef0f0.png)

Y la verdad que ya no tenemos tiempo, preferible hacer el refactor despues con mas margen